.github/workflows/crowdin.yml

```
@@ -0,0 +1,38 @@
+          upload_translations: false
+          download_translations: true
+          localization_branch_name: l10n_crowdin_translations
+          create_pull_request: true
+          pull_request_title: 'New Crowdin Translations'
+          pull_request_body: 'New Crowdin translations by [Crowdin GH Action](https://github.com/crowdin/github-action)'
+          pull_request_base_branch_name: 'main'
+          config: '.crowdin.yml'
+        env:
+          # A classic GitHub Personal Access Token with the 'repo' scope selected (the user should have write access to the repository).
+          GITHUB_TOKEN: ${{ secrets.CROWDIN_TOKEN }}
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)